### ISSUE TITLE
With Java 10 Tomcat will not Start

### ISSUE THREAD

**Issue by pfreitag** [viviotech/lucee-installer#94]
When I switch the JVM to Java 10 (I am using 10.0.2) the Tomcat Service does not start. It turns out it is because the system property `java.endorsed.dirs` is not supported on Java 10.
`-Djava.endorsed.dirs=C:\lucee\tomcat\endorsed is not supported. Endorsed standards and standalone APIs in modular form will be supported via the concept of upgradeable modules.`

I wrote up the details here: https://www.petefreitag.com/item/851.cfm

**Comment by utdream** [comment]
Awesome. Thanks Pete. Very helpful!

**Comment by utdream** [comment]
Notes on this:

This specific issue looks to have been addressed by rjung in Tomcat 8.5.24, but I failed to implement it until now. Using the updated catalina[.sh|.bat] scripts from this release on (there have been a couple fixes to it in newer releases) Tomcat will now properly ignore the "endorsed" aspect of the JRE if it's not supported.

I performed a successful test of running Oracle's OpenJDK 11.0.1 on Lucee 5.3 RC and with the new changes from the updated catalina.sh, things can proceed largely unchanged, but more work is needed to polish it and make it nice. For example, there is a lot of excessive output in various logs that doesn't need to be there. A SystemD script would be particularly helpful for modern releases of CentOS and Ubuntu... See issue 93.
